2021/webauthn-2021.html

```diff
@@ -166,6 +166,7 @@ <h2>Scope</h2>
 		  <li> Dynamic linking of authentication credentials; 
 		  <li> Storing of private key(s); 
 		  <li> Account recovery and/or credential backup options.
```

Comment:
Maybe the `.` here "needs" to change to `;`? @wseltzer

Reply:
thanks @emlun, fixed